cmd/kubeapps-apis/proto/kubeappsapis/plugins/kapp_controller/packages/v1alpha1/kapp_controller.proto

```diff
@@ -4,20 +4,78 @@ option go_package = "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/
 
 import "google/api/annotations.proto";
 import "kubeappsapis/core/packages/v1alpha1/packages.proto";
+import "kubeappsapis/core/plugins/v1alpha1/plugins.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 
 service KappControllerPackagesService {
-  // GetAvailablePackages returns the available packages managed by the 'kapp_controller' plugin
-  rpc GetAvailablePackages(kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesResponse) {
+  // GetAvailablePackageSummaries returns the available packages managed by the 'kapp_controller' plugin
+  rpc GetAvailablePackageSummaries(kubeappsapis.core.packages.v1alpha1.GetAvailablePackageSummariesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackageSummariesResponse) {
     option (google.api.http) = {
-      get: "/plugins/kapp_controller/packages/v1alpha1/packages"
+      get: "/plugins/kapp_controller/packages/v1alpha1/availablepackagesummaries"
     };
   };
   
   // GetPackageRepositories returns the repositories managed by the 'kapp_controller' plugin
-  rpc GetPackageRepositories(kubeappsapis.core.packages.v1alpha1.GetPackageRepositoriesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetPackageRepositoriesResponse) {
+  rpc GetPackageRepositories(GetPackageRepositoriesRequest) returns (GetPackageRepositoriesResponse) {
     option (google.api.http) = {
       get: "/plugins/kapp_controller/packages/v1alpha1/packagerepositories"
     };
   };
 }
+
+// Specific messages used by the 'kapp_controller' plugin
```

Comment:
Thanks for moving them in here. Even if we delete them soon it's nice for that to happen when the implementation is deleted, rather than with this PR.